scripts/miniapps/SNAP.md

```diff
@@ -0,0 +1,136 @@
+   ```
+   #FORTRAN = mpif90
+   #For Gnu and Intel compilers, mpifort works
+   FORTRAN = mpifort
+   ```
+
+   Select one target. When using GNU, choose `gsnap`; when using Intel
+   for KNL, choose `ksnap`; when using Intel for non-KNL choose `isnap`.
+   ```
+   TARGET = gsnap
+   #TARGET = isnap
+   #TARGET = ksnap
+   ```
+
+   If KNL is targeted, specify automatic cpu dispatch so that
+   executable can run on KNL and non-KNL.
+   ```
+   #FFLAGS = -O3 $(OMPFLAG) -xmic-avx512 -ip -align array64byte -qopt-streaming-cache-evict=0 -qno-opt-dynamic-align -fp-model fast -fp-speculation fast -fno-alias -fno-fnalias
+   FFLAGS = -O3 $(OMPFLAG) -axMIC-AVX512 -ip -align array64byte -qopt-streaming-cache-evict=0 -qno-opt-dynamic-align -fp-model fast -fp-speculation fast -fno-alias -fno-fnalias
+   ```
+   
+4. Build
+   ```
+   % make
+   ```
+
+   This creates the above selected executable in the same directory.
+
+### Example runs
+
+For each example, the number of nodes used can be varied depending on
+whether Xeon or KNL nodes are used. SNAP requires an input deck which
+matches the job launch command line with respect to the number of
+ranks etc.  Thus is is not trivial to modify number of ranks, threads
+etc. to be used. The user also has to specify an output file which
+will contain, among other things, performance data.  See
+[Caveats and other notes](#caveats-and-other-notes) below for more information on how to change
+these parameters.
+
+The following examples refer to inputs files available in this repository.
+
+To run small sanity check:
+```
+% export OMP_NUM_THREADS=2
+% export OMP_WAIT_POLICY=passive
+% srun -n 6 -N 2 --ntasks-per-node=3 -c 4 --threads-per-core 2 --hint=multithread --exclusive <path/exec> 6MT.input <outputfile>
+```
+or 
+```
+% export OMP_NUM_THREADS=2
+% aprun -n 6 -N 3 -d 2 -cc depth -j 2 <path/exec> 6MT.input <outputfile>
+```
+
+To run NERSC small problem on 96 ranks and 4 nodes:
+```
+% export OMP_NUM_THREADS=1
+% srun -n 96 -N 4 --ntasks-per-node=24 --threads-per-core 1 --exclusive <path/exec> 96ST4nodes.input <outputfile>
+```
+   or
+```
+% export OMP_NUM_THREADS=1
+% aprun -n 96 -N 24 <path/exec> 96ST4nodes.input  <outputfile>
+```
+
+To run problem on 512 ranks:
+```
+% export OMP_NUM_THREADS=4   # must match input deck, see below
+% export OMP_WAIT_POLICY=passive
+% srun -n 512 -N 8 --ntasks-per-node=64 -c 4 --threads-per-core 4 --hint=multithread --exclusive <path/exec> 512MT.input <outputfile>
+```
+   or
+```
+% export OMP_NUM_THREADS=4   # must match input deck, see below
+% export OMP_WAIT_POLICY=passive
+% aprun -n 512 -N 64 -d 4 -cc depth -j 4 <path/exec> ./512MT.input <outputfile>
+```
+
+where `<path/exec>` refers to `gsnap`, `ksnap`, or `isnap`.
+
+## Expected output
+
+For each run, look for "Success! Done in a SNAP!" on stdout to confirm
+completion.
+
+Performance data is summarized at the end in `<outputfile>. In
+particular look for "Solve" time and "Grind Time".  Solve time is the
```

Comment:
Should the '`' be removed?
When looking at this information through the "View" file option, "<outputfile>" does not appear.  All you see is the '`'.